mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -930,7 +930,7 @@ void CMomentumGameMovement::FinishDuck(void)
     {
         Vector org = mv->GetAbsOrigin();
 
-        if (player->GetGroundEntity() != nullptr)
+        if (player->GetGroundEntity() != nullptr || m_pPlayer->m_CurrentSlideTrigger)
```

Comment:
How is this related to the replay stuff?

Reply:
I let it slide after talking about it in the game code channel on Discord. They know for the future that small fixes like this should be in their own branch.